### PR TITLE
Feature/allow upgrade step to disable themselves

### PR DESCRIPTION
**Pull request by firone**
Can be merged

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -65,7 +65,7 @@
     <logstash-logback.version>4.5.1</logstash-logback.version>
     <metrics-spring.version>3.0.0</metrics-spring.version>
     <nosqlunit.version>0.7.9</nosqlunit.version>
-    <obiba-commons.version>1.8.4</obiba-commons.version>
+    <obiba-commons.version>1.8.5</obiba-commons.version>
```
Comment (ymarcon) [inline pom.xml]: don't you need features from 1.9-SNAPSHOT?

Comment (firone) [inline pom.xml]: There is no difference between 1.8.x and master.
I don't know why these 2 branches are separated ...

Comment (firone) [inline pom.xml]: Changes about UpgradeStep are in master (1.9-SNAPSHOT) and 1.8.x

Comment (ymarcon) [inline pom.xml]: You are not supposed to add new features in a branch.

**Pull request merged**